src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -60,6 +61,10 @@ class ApplicationMenu extends Component<Props> {
     shell.openExternal(`${GUPPY_REPO_URL}/${pathname}`);
   };
 
+  openLink = url => {
+    shell.openExternal(url);
+  };
```

Comment:
Nit: Personal opinion is that this abstraction doesn't add a ton of value =) I'd just call `shell.openExternal` directly.

But yeah that's a personal preference thing, so no changes necessary.

Reply:
yes, you're right. That's not needed. I'll change it and inline it.